recipes/FC-Virus/mete.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: FC-Virus
+  version: "1.0.0"
+  noarch: generic
```

Comment:
This should be moved under 'build'

Reply:
Thanks for your reply. I've moved the relevant section under 'build' as suggested.